bouncer/scripts/test/redirect-test.js

```diff
@@ -71,6 +72,40 @@ describe('redirect()', function () {
       true);
   });
 
+  it('redirects to Via if lastError is defined', function () {
+    window.chrome = {
+      runtime: {
+        sendMessage: function (id, message, callbackFunction) {
+          callbackFunction('Hey!');
+        },
+        lastError: 'There was an error'
```

Comment:
This doesn't match the actual structure of errors returned by Chrome, which is `{message: string}`, I suggest that we use a fake error which has the same structure at least.